src/ui/hooks/useMarkdown.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {YfmWrapper} from '../components/YfmWrapper/YfmWrapper';
+import {DL} from '../constants';
+import {getSdk} from '../libs/schematic-sdk';
+
+type Props = {
+    value: string;
+};
+
+const MarkdownCollection = new Map();
```

Comment:
Possible memory leak. Maybe WeakMap at least?

Reply:
There can be no memory leak here, since the point is just to keep the downloaded data in memory

In order not to overflow the memory limit and the user interface remained working, added a limit on the cache size - 1k